packages/mobile/src/screens/collection-screen/CollectionScreen.tsx

```diff
@@ -55,7 +55,8 @@ const useStyles = makeStyles(({ spacing }) => ({
 export const CollectionScreen = () => {
   const { params } = useRoute<'Collection'>()
 
-  const { id, searchCollection } = params
+  // params is incorrectly typed and can sometimes be undefined
+  const { id, searchCollection } = params ?? {}
```

Comment:
just checking for sanity, but when these are undefined, we're guarded below on the !collection || !user check

Reply:
yeah exactly, we just dont get user or collection and return null